File: python/prism_sdk/federated_continual_research_bundle_integrity_inference.py
"""Bundle P32 federated_continual inference signed research-object integrity feature."""
from .research_bundle_integrity_support import BundleCard7,BundleReleaseRequest4,ResearchBundleIntegrityError,manifest,release
FEATURE_ID="AFA-bundle-P32-F04";CONTRACT_VERSION="bundle-federated_continual_research_bundle_integrity_inference/1.0"
def federated_continual_research_bundle_integrity_inference_manifest():return manifest(feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="federated_continual",mode="inference")
def release_federated_continual_research_bundle_integrity_inference(request:BundleReleaseRequest4)->BundleCard7:return release(request,feature_id=FEATURE_ID,contract_version=CONTRACT_VERSION,scale="federated_continual",mode="inference")
__all__=["FEATURE_ID","CONTRACT_VERSION","federated_continual_research_bundle_integrity_inference_manifest","release_federated_continual_research_bundle_integrity_inference"]

File: crates/bundle/src/federated_continual_research_bundle_integrity_inference.rs
//! Bundle P32 federated_continual inference signed research-object integrity feature.
use super::research_bundle_integrity_support::{manifest,release,BundleCard7,BundleReleaseRequest4,ResearchBundleIntegrityError};
pub const FEATURE_ID:&str="AFA-bundle-P32-F04";pub const CONTRACT_VERSION:&str="bundle-federated_continual_research_bundle_integrity_inference/1.0";
pub fn federated_continual_research_bundle_integrity_inference_manifest()->serde_json::Value{manifest(FEATURE_ID,CONTRACT_VERSION,"federated_continual","inference")}
pub fn release_federated_continual_research_bundle_integrity_inference(request:&BundleReleaseRequest4)->Result<BundleCard7,ResearchBundleIntegrityError>{release(request,FEATURE_ID,CONTRACT_VERSION,"federated_continual","inference")}
